app/models/person.rb

```diff
@@ -95,6 +95,7 @@ class Person < ApplicationRecord
 
   has_many :listings, -> { exist }, :dependent => :destroy, :foreign_key => "author_id"
   has_many :working_time_slots, :through => :listings, :class_name => "Listing::WorkingTimeSlot"
+  has_many :bookings_per_hour, ->{ per_hour_blocked }, :through => :listings
```

Comment:
Kan der holdes en eller anden form for :abc: her? Den skal formentligt være efter `listings`, ikke, hvilket gør det svært?

Reply:
Den skal være efter listings, men vi kan godt sætte den før `:working_time_slots`, så er der lidt sortering over det